src/components/distributions/editor/TextForm/TextForm.js

```diff
@@ -63,6 +68,13 @@ export class TextForm extends Component{
       m => !!_.get(m, 'simulation.sample.errors.length') || !_.get(m, 'simulation.sample.values.length')
     ).map(m => m.readableId)
 
+    // To see if this guesstimate is a valid choice for a lognormal distribution, we'll try to parse it with
+    // guesstimateType manually set to 'LOGNORMAL', and see if the parser corrects that type to something else. This
+    // approach is a bit hacky, but it gets the job done.
+    const [_1, parsed] = Guesstimator.parse({input, guesstimateType: 'LOGNORMAL'})
+    const parsedType = _.get(parsed, 'parsedInput.guesstimateType')
+    const isLognormalValid = parsedType === 'LOGNORMAL'
```

Comment:
hmm, I'd kind of rather leave it here, and just put it into the Guesstimator/Parser later when we refactor those. This is a definite hack right now, so I'd rather not cement it in a library if that makes sense.